Documentation meta-descriptions review N° 4 (#5558)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/verifying-configuration.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: Find out how to verify several types of configurations. Learn more about it in this section of the Wazuh documentation.

Reviewer: ```suggestion
  :description: Find out how to verify sections of the Wazuh configuration in this section of the Wazuh documentation.
```
